src/nycdb/datasets/nycha_bbls.yml

```diff
@@ -0,0 +1,21 @@
+---
+files:
+  -
+    url: https://raw.githubusercontent.com/JustFixNYC/nycha-scraper/master/Block-and-Lot-Guide-08272018.csv
```

Comment:
The one thing I'm a _little_ worried about here is just that this file wasn't originally intended to be permalinked-to, so it might change at some point, and then this link would break.

One way to fix this is by referring to that file at a specific commit hash, e.g.:

https://raw.githubusercontent.com/JustFixNYC/nycha-scraper/098bd8232bee2cd59266bf278e26e32bd0cd7df1/Block-and-Lot-Guide-08272018.csv

That's the latest version of the file at https://github.com/JustFixNYC/nycha-scraper/commit/098bd8232bee2cd59266bf278e26e32bd0cd7df1, and if for some reason we decide to remove it from the repository, or rename it, or something else, nycdb won't break.  The downside is that if we ever somehow improve on the dataset (e.g. we find a bug in the scraping logic and re-scrape the PDF) we'd have to remember to also update the revision that this URL points to.  But I feel like that's less risky than accidentally breaking NYCDB.

Reply:
Good point! Ok I just updated the link.